transport/grpc/dial.go

```diff
@@ -261,15 +259,7 @@ func isDirectPathEnabled(endpoint string) bool {
 		return false
 	}
 
-	// Only try direct path if the user has opted in via the environment variable.
-	directPathAPIs := strings.Split(os.Getenv("GOOGLE_CLOUD_ENABLE_DIRECT_PATH"), ",")
```

Comment:
This seems like a breaking change for users. I think if we want to support something like this feature we still need to support the old way.

Reply:
This should be fine since the environment variable is only used internally. More background are provided in email. Thanks for catching!